metrics/lib/common.bash

```diff
@@ -65,6 +65,26 @@ function check_cmds()
 	done
 }
 
+# A one time (per uber test cycle) init that tries to get the
+# system to a 'known state' as much as possible
+function onetime_init()
+{
+	# The onetime init must be called once, and only once
+	if [ ! -z "$onetime_init_done" ]; then
+		die "onetime_init() called more than once"
```

Comment:
Should we die, or just warn and return?

Reply:
Hi @bwarden . My theory is that if somebody is calling this more than once then they are potentially heading back to the situation we have right now, where we are calling the restart frequently, which upsets systemd. I'd rather enforce that it is only called once, and hence the `die`. A `warn` I suspect will just go ignored in the logs.

Reply:
Ah, and I see what you mean - should we 'just warn and return', and be more benign. Hmm, yes, that would work, but I think I'd rather make it more blatant that somebody has done something they should not, rather than fairly benign (and if they do not spot the warning in the logs, they may continue under the misbelief that we have done something when actually we have not)

Reply:
Makes sense, lgtm.